### PR TITLE
Add CR for openebs 2.11.0

### PR DESCRIPTION
**Pull request by aamirqs**
Signed-off-by: Aamir Shaikh <aameer.shaikh@mayadata.io>
tested deployment on Openshift 4.6 OpenEBS was successfully deployed
```
root@aamir-inspiron-3593:/home/aamir/projects/helm-operator/olm# oc version
Client Version: 4.7.13
Server Version: 4.6.41
Kubernetes Version: v1.19.0+4c3480d
root@aamir-inspiron-3593:/home/aamir/projects/helm-operator/olm# 

root@aamir-inspiron-3593:/home/aamir# oc get pods
NAME                                                READY   STATUS    RESTARTS   AGE
oebs-openebs-admission-server-548794cff5-5gs95      1/1     Running   0          7m15s
oebs-openebs-apiserver-5d6648497c-4thsl             1/1     Running   0          7m15s
oebs-openebs-localpv-provisioner-7cc7464556-gz4cn   1/1     Running   0          7m15s
oebs-openebs-ndm-operator-7cdd54584f-xb7s6          1/1     Running   0          7m15s
oebs-openebs-provisioner-8dbbdcc5f-v5786            1/1     Running   0          7m15s
oebs-openebs-snapshot-operator-559498b555-mngtw     2/2     Running   0          7m15s
openebs-helm-operator-798f58bfb-p4wsb               1/1     Running   0          7h32m
root@aamir-inspiron-3593:/home/aamir# oc get sc
NAME                        PROVISIONER                                                RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
gp2 (default)               kubernetes.io/aws-ebs                                      Delete          WaitForFirstConsumer   true                   4d16h
gp2-csi                     ebs.csi.aws.com                                            Delete          WaitForFirstConsumer   true                   4d16h
openebs-device              openebs.io/local                                           Delete          WaitForFirstConsumer   false                  4d6h
openebs-hostpath            openebs.io/local                                           Delete          WaitForFirstConsumer   false                  4d6h
openebs-jiva-default        openebs.io/provisioner-iscsi                               Delete          Immediate              false                  4d6h
openebs-snapshot-promoter   volumesnapshot.external-storage.k8s.io/snapshot-promoter   Delete          Immediate              false                  4d6h

```

### CODE REVIEW

File: deploy/crds/openebs_v1alpha1_openebsinstalltemplate_cr.yaml [hunk 1]
```diff
@@ -1,146 +1,153 @@
...
       periodSeconds: 60
-    nodeSelector: {}
-    tolerations: []
-    affinity: {}
     hostNetwork: false
-
+  helper:
+    image: quay.io/openebs/linux-utils
+    helperImageTag: 2.11.0
+  featureGates:
```
Comment (kmova) [inline deploy/crds/openebs_v1alpha1_openebsinstalltemplate_cr.yaml:123]: Is this part of NDM config?

Comment (aamirqs) [reply deploy/crds/openebs_v1alpha1_openebsinstalltemplate_cr.yaml:123]: @kmova I checked it from https://github.com/openebs/charts/blob/9beb1dd261939fe22d758bbfa9b0b1814d6d3d87/charts/openebs/values.yaml#L250 the helper and feature gates are not under ndm operator but at same level

**Review by kmova — APPROVED** [review]

File: olm/2.11.0/openebsoperator.v2.11.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,575 @@
...
+              "enabled": true,
+              "image": "quay.io/openebs/m-apiserver",
+              "imageTag": "2.11.0",
+              "replicas": 1,
+              "ports": {
+                "externalPort": 5656,
+                "internalPort": 5656
+              },
+              "sparse": {
+                "enabled": "false"
```
Comment (nareshdesh) [inline olm/2.11.0/openebsoperator.v2.11.0.clusterserviceversion.yaml:53]: remove quotes around false

File: olm/2.11.0/openebsoperator.v2.11.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,575 @@
...
+              },
+              "healthCheck": {
+                "initialDelaySeconds": 30,
+                "periodSeconds": 60
+              }
+            },
+            "defaultStorageConfig": {
+              "enabled": "true"
```
Comment (nareshdesh) [inline olm/2.11.0/openebsoperator.v2.11.0.clusterserviceversion.yaml:61]: remove quotes around true

**Pull request merged**